nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -513,6 +513,8 @@ private String buildCommitMsg(TableMetadata base, TableMetadata metadata, String
           "Iceberg %s against %s", metadata.currentSnapshot().operation(), tableName);
     } else if (base != null && metadata.currentSchemaId() != base.currentSchemaId()) {
       return String.format("Iceberg schema change against %s", tableName);
+    } else if (base == null) {
+      return String.format("Iceberg table registered with name %s", tableName);
```

Comment:
Does create table also enter this flow? 🤔

Reply:
Yes it does, sharing an example effect of this change: 

![Screenshot 2023-08-25 at 9 38 28 AM](https://github.com/apache/iceberg/assets/57615655/57e7c6b1-e2c5-4235-ba48-1d959f34b5fb)

Reply:
So, we can't differentiate create table and register flow? 

If there is no way, I am ok with having this commit message (which was better than previous)

Reply:
IMO `registered` has a special meaning, I'd favor `created/registered` instead of just `registered`

Reply:
+1 to @snazy 's suggesion

Reply:
Thanks, changed the message likewise `created/registered instead of just registered`.

> So, we can't differentiate create table and register flow?
The nearest possibility is to consider the case as `create` if there are no snapshots in the new metadata. Since it may not always be true, not pursuing that path.